graylog2-server/src/main/resources/prometheus-exporter.yml

```diff
@@ -0,0 +1,1344 @@
+    additional_labels:
+      type: "submitted"
+
+  - metric_name: "system_job_manager_executor_duration"
+    match_pattern: "org.graylog2.system.jobs.SystemJobManager.executor-service.duration"
+
+  - metric_name: "system_job_manager_executor_scheduled_once"
+    match_pattern: "org.graylog2.system.jobs.SystemJobManager.executor-service.scheduled.once"
+
+  - metric_name: "system_job_manager_executor_scheduled_overrun"
+    match_pattern: "org.graylog2.system.jobs.SystemJobManager.executor-service.scheduled.overrun"
+
+  - metric_name: "system_job_manager_executor_scheduled_percent-of-period"
+    match_pattern: "org.graylog2.system.jobs.SystemJobManager.executor-service.scheduled.percent-of-period"
+
+  - metric_name: "system_job_manager_executor_scheduled_repetitively"
+    match_pattern: "org.graylog2.system.jobs.SystemJobManager.executor-service.scheduled.repetitively"
+
+  - metric_name: "input_throughput"
+    match_pattern: "org.graylog2.throughput.input"
+
+  - metric_name: "output_throughput"
+    match_pattern: "org.graylog2.throughput.output"
+
+  - metric_name: "traffic_decoded"
+    match_pattern: "org.graylog2.traffic.decoded"
+
+  - metric_name: "traffic_input"
+    match_pattern: "org.graylog2.traffic.input"
+
+  - metric_name: "traffic_output"
+    match_pattern: "org.graylog2.traffic.output"
+
+  - metric_name: "traffic_system_output_traffic"
+    match_pattern: "org.graylog2.traffic.system-output-traffic"
+
+  - metric_name: "forwarder_grpc_method_invocations"
+    match_pattern: "org.graylog.plugins.cloud.forwarder.grpc.ServerMetricsInterceptor.method.org.graylog.cloud.forwarder.*.invocations"
+    wildcard_extract_labels:
+      - "method"
+
+  - metric_name: "forwarder_grpc_method_executionTime"
+    match_pattern: "org.graylog.plugins.cloud.forwarder.grpc.ServerMetricsInterceptor.method.org.graylog.cloud.forwarder.*.executionTime"
+    wildcard_extract_labels:
+      - "method"
+
+  - metric_name: "auth_actions"
+    match_pattern: "org.graylog.plugins.cloud.auth.*.success"
+    wildcard_extract_labels:
+      - "action"
+    additional_labels:
+      success: "true"
+
+  - metric_name: "auth_actions"
+    match_pattern: "org.graylog.plugins.cloud.auth.*.failure.http.*"
+    wildcard_extract_labels:
+      - "action"
+      - "http_code"
+    additional_labels:
+      success: "false"
+      type: "http"
+
+  - metric_name: "auth_actions"
+    match_pattern: "org.graylog.plugins.cloud.auth.*.failure.okta.*"
+    wildcard_extract_labels:
+      - "action"
+      - "okta_error_code"
+    additional_labels:
+      success: "false"
+      type: "okta"
+
+  - metric_name: "auth_actions"
+    match_pattern: "org.graylog.plugins.cloud.auth.*.failure.gl.*"
+    wildcard_extract_labels:
+      - "action"
+      - "graylog_error_code"
+    additional_labels:
+      success: "false"
+      type: "graylog"
+
+  - metric_name: "input-buffer-thread-factory"
```

Comment:
Shouldn't these be underscores instead?

Reply:
@fungusakafungus You are absolutely right. Thank you very much for seeing this. https://github.com/Graylog2/graylog2-server/issues/10904 has been created, and we will fix this.

Reply:
This has been fixed in https://github.com/Graylog2/graylog2-server/pull/10905 and will be included in the 4.1 release.